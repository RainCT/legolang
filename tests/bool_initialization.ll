// Bools are false by default
bool b
print b

// And we can assign to them
b = true
print b

b = false
print b

b = true == false
print b

bool c
print "Another bool: " + c
