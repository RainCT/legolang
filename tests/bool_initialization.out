false
true
false
false
Another bool: false
